src/analyzer.ts

```diff
@@ -98,6 +98,10 @@ export class Analyzer {
     return this._context.load(resolvedUrl, providedContents);
   }
 
+  canResolveUrl(url: string): string {
```

Comment:
Consider leaving this out to keep Analyzer as simple as possible (unless there's a use case for it).

Also, I'm surprised that this type checks, as canResolveUrl returns a boolean